documentation/src/main/asciidoc/user_guide/clustering.adoc

```diff
@@ -39,6 +39,9 @@ nodes; reads are local.
 * *Distributed*: changes are replicated to a fixed number of nodes, reads request the
 value from at least one of the owner nodes.
 
+//dnaro: filtering this table from product docs. RH doc tooling does not
+//support asciimath or text coloring. Including this table throws a warning.
+ifndef::productized[]
```

Comment:
What's the plan for including this in the product in the future instead of just ignoring? It's a useful table that would be good to include.

Reply:
Hey Ryan, agreed that the table is useful. It contains an equation for calculating the cache size or something like that, which requires asciimath to render correctly. RH docs tooling doesn't support asciimath though so the equation just renders as asciidoc markup, which doesn't make any sense and negates the benefit for users.
I'm trying to find out if we can get support for asciimath, but probably not. Otherwise I think maybe we can include the equation as an image and re-format the table. This is definitely on my to-do list and I don't want to just ignore it in the docs.

Reply:
Sounds good. An image seems like a good backup if need be.